python/lsst/ip/isr/ampOffset.py

```diff
@@ -457,8 +485,9 @@ def getInterfaceOffset(self, ampIdA, ampIdB, edgeA, edgeB):
             interfaceOffset = 0
```

Comment:
Perhaps, under this `if` statement, there should be separate `warning` messages for the case of either `minFracFail` or `maxOffsetFail`. The warning message is getting quite long now, which probably warrants it being split up into two.